config.template.js

```diff
@@ -11,6 +11,9 @@ const CONFIG = {
     socket: false,
     socketPath: '/tmp/rtorrent.sock'
   }
+  ssl: false,
+  sslKey: '/etc/letsencrypt/live/my.awesome.domain.com/fullchain.pem',
+  sslCert: '/etc/letsencrypt/live/my.awesome.domain.com/privkey.pem'
```

Comment:
Let's use a more generic and obviously false pathname like `/absolute/path/to/key/` and `/absolute/path/to/certificate/`